File: app/src/main/java/xlfdll/nb/fpg/AppHelper.java
package xlfdll.nb.fpg;

import android.content.Context;
import android.content.DialogInterface;
import android.content.SharedPreferences;
import android.support.v7.app.AlertDialog;

/**
 * Created by Xlfdll on 2017/08/31.
 */

class AppHelper {
    public static SharedPreferences Settings;

    public static void showMessageDialog(Context context, String title, String message) {
        AlertDialog.Builder builder = new AlertDialog.Builder(context);

        builder.setMessage(message)
                .setTitle(title)
                .setPositiveButton("OK", new DialogInterface.OnClickListener() {
                            public void onClick(DialogInterface dialog, int whichButton) {
                                dialog.dismiss();
                            }
                        }
                )
                .create()
                .show();
    }

    public static void showMessageDialog(Context context, int titleId, int messageId) {
        AlertDialog.Builder builder = new AlertDialog.Builder(context);

        builder.setMessage(messageId)
                .setTitle(titleId)
                .setPositiveButton("OK", new DialogInterface.OnClickListener() {
                            public void onClick(DialogInterface dialog, int whichButton) {
                                dialog.dismiss();
                            }
                        }
                )
                .create()
                .show();
    }
}

File: app/src/main/java/org/xlfdll/nb/fpg/AppHelper.kt
package org.xlfdll.nb.fpg

import android.content.Context
import android.content.DialogInterface
import android.content.SharedPreferences
import androidx.appcompat.app.AlertDialog

/**
 * Created by Xlfdll on 2017/08/31.
 */

internal object AppHelper {
    internal lateinit var Settings: SharedPreferences

    internal fun showMessageDialog(context: Context, title: String, message: String) {
        val builder = AlertDialog.Builder(context)

        builder.setMessage(message)
                .setTitle(title)
                .setPositiveButton("OK"
                ) { dialog, _ ->
                    dialog.dismiss()
                }
                .create()
                .show()
    }

    internal fun showMessageDialog(context: Context, titleId: Int, messageId: Int) {
        val builder = AlertDialog.Builder(context)

        builder.setMessage(messageId)
                .setTitle(titleId)
                .setPositiveButton("OK"
                ) { dialog, _ ->
                    dialog.dismiss()
                }
                .create()
                .show()
    }
}